examples/api-key-management/Ballerina.toml

```
@@ -0,0 +1,15 @@
+[package]
+org = "yasith_rashan"
```

Use the `wso2` organization for the examples. 

```suggestion
org = "wso2"
```